src/test/java/de/fraunhofer/aisec/cpg/frontends/cpp/CXXIncludeTest.java

```diff
@@ -88,4 +90,114 @@ void testCodeAndRegionInInclude() throws Exception {
 
     assertEquals(new Region(16, 3, 16, 15), location.getRegion());
   }
+
```

Comment:
Propose to add a test case that tests a file contained in white- *and* blacklist as this is what will most likely create confusion/error cases.

Reply:
I added a test case (only for relative paths though).